apps/rush/src/start.ts

```diff
@@ -36,19 +38,77 @@ const configuration: MinimalRushConfiguration | undefined =
 
 const currentPackageVersion: string = PackageJsonLookup.loadOwnPackageJson(__dirname).version;
 
-let rushVersionToLoad: string | undefined = undefined;
+let rushVersionToLoadInfo: {
+  version: string;
+  path?: string;
+} | undefined = undefined;
+
+let overrideInfo: {
+  version: string;
+  path: string;
+} | undefined = undefined;
+
+if (configuration) {
+  const rushOverrideFilePath: string = path.join(
+    path.dirname(configuration.rushJsonFilename),
+    rushLib.RushConstants.commonFolderName,
+    rushLib.RushConstants.rushTempFolderName,
+    '.rush-override'
+  );
+
+  if (fs.existsSync(rushOverrideFilePath)) {
```

Comment:
Use `FileSystem` from `node-core-library`, and do a try-catch on reading the file and gracefully handle it not existing. See `FileSystem.isNotExistError`